### ISSUE TITLE
pyproject example

### ISSUE THREAD

**Issue by MartinThoma** [pchomik/pytest-spec#56]
Is there a recommended configuration for pyproject? When I install pytest-spec, nothing seems to change. When I add

```
[tool.pytest]
spec_header_format = "{module_path}:"
spec_test_format = "{result} {name}"
```

to my `pyproject.toml`, nothing seems to change either.

**Comment by jeffnyman** [comment]
For what it's worth I had to make the group the following:

```
[tool.pytest.ini_options]
addopts = "--spec"
spec_test_format = "{result} {docstring_summary}"
```

That addition of "ini_options" is what ended up working for me.

**Comment by ya55en** [comment]
Would be great to have this in the README as well ... I would submit a PR if that's acceptable.

**Comment by pchomik** [comment]
Solved by #58

**Comment by kassi** [comment]
I have the same problem but even the addition of `addopts = "--spec"` didn't help.
Anything else to do?
I'm having a simple poetry created module, pytest, pytest-spec.
```
poetry run pytest -vv
```
looks the same as before without pytest-spec.
